server/src/main/res/layout/activity_connect_to_client.xml

```diff
@@ -10,7 +10,7 @@
         android:id="@+id/textView"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="140dp"
+        android:layout_marginTop="200dp"
```

Comment:
why did you change the margin? It seems unrelated to the topic of this pull request.
I am worried that if the margin is too big it might cause problems on devices with a smaller screen.